src/nunit.analyzers.tests/TestUtility.cs

```diff
@@ -1,9 +1,7 @@
-using System.Linq;
-using System.Threading.Tasks;
 using Gu.Roslyn.Asserts;
 using NUnit.Framework;
 
-[assembly: MetadataReferences(typeof(Assert), typeof(object), typeof(Task), typeof(Enumerable))]
```

Comment:
Previously most tests failed with issues reported by `Gu.Roslyn.Asserts` similar to:
>  CS0012 The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.

Let's be honest, I have no idea why it failed, and why this change fixed that.

Reply:
As far as I can remember one needs to mention the classes used via `MetadataReferences`, so perhaps a `typeof(Object)` would also have solved the issue (although it seems from https://github.com/dotnet/standard/issues/542#issuecomment-530451163 that this is a general problem). @JohanLarsson can probably answer it better than I.

That being said the README of Gu.Roslyn.Asserts - https://github.com/GuOrg/Gu.Roslyn.Asserts/blob/master/README.md -  states `Below should not be needed (i.e. adding MetadataReferences), prefer TransitiveMetadataReferences as it updates as the resolution graph changes.`, so I think your solution is sound.